### PR TITLE
fix: self.installed_versions is not being handled properly

### PR DESCRIPTION
**Pull request by sabotagebeats**
fixed as per https://github.com/ApeWorX/ape-vyper/issues/17#issuecomment-876689353 & tested working locally

### CODE REVIEW

**Review by fubuloubu — COMMENTED** [review]
Can you add another test contract, with this one just being empty (no version pragma) to test this behavior? I think that should fail without this fix.

**Comment by sabotagebeats** [comment]
@fubuloubu do I need to add that contract in the test_integration.py as well?

**Comment by fubuloubu** [comment]
> @fubuloubu do I need to add that contract in the `test_integration.py` as well?

yup, although it should still just compile and trip the bug as well.

looks like we are seeing the issue when it's unspecified of which version to use!

File: ape_vyper/compiler.py [hunk 1]
```diff
@@ -92,6 +92,8 @@ def compile(self, contract_filepaths: List[Path]) -> List[ContractType]:
                     else:
                         raise Exception("No available version to install")
             else:
+                if max(self.installed_versions) is None:
+                    vvm.install_vyper(max(self.available_versions), show_progress=True)
```
Comment (sabotagebeats) [inline ape_vyper/compiler.py:96]: https://github.com/ApeWorX/ape-vyper/pull/19#issuecomment-876875943

**Comment by sabotagebeats** [comment]
to recreate this error I needed to uninstall my vyper versions 
https://github.com/ApeWorX/ape-vyper/pull/19/checks?check_run_id=3025282666#step:5:56

File: ape_vyper/compiler.py [hunk 2]
```diff
@@ -92,7 +92,9 @@ def compile(self, contract_filepaths: List[Path]) -> List[ContractType]:
                     else:
                         raise Exception("No available version to install")
             else:
-                vyper_version = max(self.available_versions)
+                if len(self.installed_versions) == 0:
```
Comment (fubuloubu) [inline ape_vyper/compiler.py:95]: `len` check is equivalent to
```suggestion
                if not self.installed_versions:
```

**Review by fubuloubu — APPROVED** [review]
This looks great! Please update title to describe issue
